src/web/screens/home.scss

```diff
@@ -0,0 +1,64 @@
+.dashboard-cards-container {
+  display: flex;
+  gap: 22px;
+  margin-top: 28px;
+
+  & > div {
+    border: 1px solid var(--theme-border);
+    border-radius: 8px;
+    width: 487px;
```

Comment:
Generally, setting width for layout purposes is a bad idea - use max-width instead and make sure flex is working.
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/111951647/64d45269-ed07-4e53-9065-482a27c9673a)